_posts/2024-10-15-anaconda-defaults.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: post
+title: A Guide to the Anaconda `defaults` channel
+image: /assets/images/way-of-the-mambalorian.webp
+
+tags:
+  - Conda
+  - bioconda
+  - conda-forge
+  - defaults
+contributors:
+  - ammaraziz
+reviewers:
+- pmenzel
+- jfy133
+---
+
+
+![In the art style of 1990s Disney, cel shading: A mysterious blacksmith in a dark, forge-like setting. She wears a sleek, gold-plated helmet embossed with the symbol of a snake and an armored robe with intricate designs. Sparks fly as she hammers a molten weapon on an anvil, which begins to take the shape of a coiled snake. The forge’s flames cast a warm glow, reflecting off her helmet and tools. The snake is a python with a green sheen.]({% link assets/images/way-of-the-mambalorian.webp %})
```

Comment:
It's quite a large square. How would you feel about floating it? If you like the look of this I'll push the patch

Mobile | Desktop
--- |---
![Screen Shot 2024-10-25 at 09 51 53](https://github.com/user-attachments/assets/74651ef6-8697-4bf8-98f6-570fc903d5f5) | ![Screenshot 2024-10-25 at 09-51-39 A Guide to the Anaconda defaults channel µbinfie Blog](https://github.com/user-attachments/assets/ab3ca73c-5d78-4024-bc7d-3dd28a8bb961)

Reply:
That looks awesome!